app/views/checkEligibility/InEligibleLocationOtherView.scala.html

```diff
@@ -0,0 +1,40 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.checkEligibility.routes._
+@import views.ViewUtils._
+@import config.FrontendAppConfig
+
+@this(
+    govukLayout: templates.GovukLayoutWrapper,
+    h1: components.h1,
+    link: components.link,
+    p : components.p
+)
+
+@()(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@govukLayout(pageTitle = Some(titleNoForm("notEligible.title", None)), signOut = false) {
+
+    @h1(messages("notEligible.heading"))
+
+    @p(Html(messages("notEligible.locationOther.p1")))
+    @p(Html(messages("notEligible.p2")))
+    @p{
+        @messages("notEligible.p3")
+        @link("https://www.gov.uk/charities-and-tax/get-recognition",messages("notEligible.p3.link"), isExternal = true)
```

Comment:
can we move this  link to application.conf and then read it using appConfig please?